mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapper.java

```diff
@@ -92,7 +92,7 @@ public Mailbox findMailboxByPath(MailboxPath path) throws MailboxException {
                 .thenCompose(cassandraIdOptional ->
```

Comment:
Could we have a **unit test**? That's a great finding, we should not have some regressions...

(Mapper level? In MailboxMapperACLTest should be the right place for it.)